app/team.go

```diff
@@ -709,11 +709,11 @@ func (a *App) GetTeamMembersForUserWithPagination(userId string, page, perPage i
 }
 
 func (a *App) GetTeamMembers(teamId string, offset int, limit int, restrictions *model.ViewUsersRestrictions) ([]*model.TeamMember, *model.AppError) {
-	result := <-a.Srv.Store.Team().GetMembers(teamId, offset, limit, restrictions)
-	if result.Err != nil {
-		return nil, result.Err
+	members, err := a.Srv.Store.Team().GetMembers(teamId, offset, limit, restrictions)
```

Comment:
Here you can just `return a.Srv.Store.Team().GetMembers(teamId, offset, limit, restrictions)`